src/pages/akcie.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { isFuture } from 'date-fns';
+
+import { Page } from 'components/Page';
+import { FBEventsList } from 'components/FBEventsList';
+import { graphql } from 'gatsby';
+
+export default ({
+  data: {
+    allFacebookEvents: { edges: events },
+  },
+}) => (
+  <Page title="Akcie" url="/akcie">
+    <div className="font-bold text-xl mb-1">Pripravované akcie</div>
+    {events && <FBEventsList events={events} filter={(event => isFuture(new Date(event.node.start_time)))} />}
+    <div className="font-bold text-xl mb-1">Minulé akcie</div>
+    {events && <FBEventsList events={events} filter={(event => !isFuture(new Date(event.node.start_time)))} />}
+  </Page>
+);
+
+export const query = graphql`
+  {
+    allFacebookEvents {
+      edges {
+        node {
+          id
+          name
+          description
+          start_time
```

Comment:
See how to localise datetime https://github.com/ParalelnaPolisKE/paralelnapoliskosice.sk/blob/master/src/fragments.js#L11 . we might not need `date-fns` package

See https://www.gatsbyjs.org/docs/graphql-reference/#dates

Reply:
I think I'll keep using `date-fns` for exact time functionality, `formatString: "HH:mm"` is not returning correct hours (should be 9:00 and 13:00)

![Screen Shot 2019-04-06 at 19 08 20](https://user-images.githubusercontent.com/6513576/55672810-ab63ce80-589f-11e9-8982-3c2604965e09.jpg)